conduct/code_of_conduct.rst

```diff
@@ -118,12 +118,12 @@ directly, or ask the Committee for advice, in confidence.
 
 You can report issues to the PySAL Code of Conduct committee at
 pysal-conduct@googlegroups.com. Also, resolutions are announced via the
-`Code of Conduct gitter channel`_. Currently, the committee
+`Code of Conduct discord channel`_. Currently, the committee
 consists of:
 
--  Stefanie Lumnitz
+-  Wei Kang
 
-   -  stefanie.lumnitz@gmail.com
+   -  WeiKang9009@gmail.com
```

Comment:
Not sure if the email address is case sensitive. So my email address is all lower-case weikang9009@gmail.com